### ISSUE TITLE
Is GPU processing a prerequisite?

### ISSUE THREAD

**Issue by hipoglucido** [udibr/headlines#4]
Hi. First of all, thanks a lot for your work.
I am about to try your scripts and I wanted to now if I could run everything without using GPUs. I have a server with 40 CPUs. Will it run?

**Comment by udibr** [comment]
Yes it runs on CPU, especially if you already have a model and just want to do predictions. But I think you will save on your electricity bill by renting a machine with GPU on Amazon with spot price (as low as $0.08/yr) and saving the model after every iteration to S3 (so you can continue your work if you happen to lose the machine because of spot price changes.)

If you are using Theano you can configure it to use CPU for example:

```
import os
os.environ['THEANO_FLAGS'] = 'device=cpu,floatX=float32'
```

**Comment by hipoglucido** [comment]
Hello @udibr, I've reading about AWS using and I have decided to do what you said. Is there any specific AMI with pre-installed dependencies that you would recommend to run your project?

Thanks!

**Comment by udibr** [comment]
go to "Community AMIs and search for CUDA

**Comment by hipoglucido** [comment]
thanks for answering @udibr . Do you mean I should search in "[shared AMIs](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/usingsharedamis-finding.html)"? I just want to be sure (I don't see the word 'Community' there)

**Comment by udibr** [comment]
https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#LaunchInstanceWizard:

**Comment by nickrobinson** [comment]
Just in case anyone runs across this issue AWS has launched a new Deep Learning AMI with everything you need to get started running this project. 

* https://aws.amazon.com/blogs/ai/the-aws-deep-learning-ami-now-with-ubuntu/
